src/_includes/layout/page-header.html

```diff
@@ -42,4 +42,9 @@ <h2 class="page-subtitle no_toc">{{ page.subtitle }}</h2>
   </div>
   {% endif %}
 
+  {% if page.url contains "safe-upgrade-tool/" %}
+  <div class="message-banner">
+    This tool is still in ALPHA and is not yet available for all customers. Submit a Magento Support ticket for assistance and questions.  </div>
```

Comment:
Does Magento Support know that we're going to direct people to them for SUT questions?

Reply:
they normally contact support or submit a ticket to request an account for magento composer if I'm not mistaken, no?

Reply:
Should I discuss this with someone to get a proper clarification about that?, in support?, or Sandra to see if she thought about that?. This comes after the conversations I had with ENG side to clarify magento composer

Reply:
@danidelcar, please confirm messaging with Sandra.

People usually self-serve on creating a Magento account to get access keys for repo.magento.com. The only time they contact Magento Support is when they're keys don't work as expected or they don't have the proper entitlements/permissions to packages on repo.magento.com.

Reply:
In January we will open the tool and it will be available for all Commerce merchants, so we can change this phrase for: "This tool is still in **ALPHA** with limited scope. If you are a Commerce user you can download it from repo.magento.com, we are looking forward to get your feedback."

Reply:
corrected include